README.md

```diff
@@ -30,7 +30,7 @@ The library provides both synchronous (blocking) API calls, as well as asynchron
 
 ## API Overview
 
-API consumers use the `DroneCore` class to discover and manage vehicles (`System` objects). Using the `System` object plugins such as e.g. `Action`, `Telemetry`, or `Mission` can be instantiated which provide information about the state of the drone and allow to interact with it.
+API consumers use the `DronecodeSDK` class to discover and manage vehicles (`System` objects). Using the `System` object plugins such as e.g. `Action`, `Telemetry`, or `Mission` can be instantiated which provide information about the state of the drone and allow to interact with it.
 
 The links below take you to the respective header files:
 
```

Comment:
Link below needs to be fixed: [dronecore](include/dronecore.h)

Reply:
Not yet, as written in the description we should change them once the github repo has moved.